cluster/images/conformance/Makefile

```diff
@@ -56,15 +59,14 @@ endif
 
 	cd ${TEMP_DIR} && sed -i.back "s|BASEIMAGE|${BASEIMAGE}|g" Dockerfile
 
-	docker build --pull -t ${REGISTRY}/conformance-${ARCH}:${VERSION} ${TEMP_DIR}
-	rm -rf "${TEMP_DIR}"
-
-push: build
-	docker push ${REGISTRY}/conformance-${ARCH}:${VERSION}
+	DOCKER_CLI_EXPERIMENTAL=enabled docker buildx build --pull --platform linux/$(ARCH) $(OUTPUT) -t ${REGISTRY}/conformance-${ARCH}:${VERSION} ${TEMP_DIR}
 ifeq ($(ARCH),amd64)
 	docker rmi ${REGISTRY}/conformance:${VERSION} 2>/dev/null || true
-	docker tag ${REGISTRY}/conformance-${ARCH}:${VERSION} ${REGISTRY}/conformance:${VERSION}
-	docker push ${REGISTRY}/conformance:${VERSION}
```

Comment:
does anything else expect them to be in the local daemon?

Reply:
> does anything else expect them to be in the local daemon?

No, I have removed the experimental in docker daemon and there is no other setting for docker daemon.
Also, I thought it might need to set builder when using docker buildx and base image is not the base image is not scratch, but it turn out builder setting is not necessary in this build.

Reply:
i mean that the previous image build would result in the images being present in the daemon locally. some of our build scripts will e.g. export a tarball from this.

Reply:
> i mean that the previous image build would result in the images being present in the daemon locally. some of our build scripts will e.g. export a tarball from this.

Yes, it will present locally. I set output format as "--load" which will present image in local daemon and can be seen via "docker image list", saved as tar via "docker save".

Reply:
please resolve this thread :upside_down_face:  :+1: